lib/rules/validate-jsdoc/enforce-existence.js

```diff
@@ -16,7 +16,8 @@ enforceExistence.configure = function(options) {
         all: true,
         anonymous: false,
         exports: true,
-        expressions: true
+        expressions: true,
+        noParamsAndReturns: true
```

Comment:
I'd add a comma ;-)

Looking for a name, what's your case? Can it be called "handlers"? Another idea: `'paramless-procedures'` (cuz procedure is a function without result).